src/main/java/org/jabref/gui/libraryproperties/LibraryPropertiesDialogView.java

```diff
@@ -33,6 +34,7 @@
 
     @FXML private VBox contentVbox;
     @FXML private ComboBox<Charset> encoding;
+    @FXML private ComboBox<String> databaseMode;
```

Comment:
If you use `ComboBox<BibDatabaseMode>` here (and propagate the change), then you don't have to constantly convert between formatted values and parsed modes.

Reply:
If I do so, the names are not formatted anymore. Also, the view would have knowledge about the model.
I normally always try to avoid that by converting the model to non model datatypes. So I would suggest to leave it this way, what do you think?

Reply:
You can get the proper display quite easily using similar code as the one below:
https://github.com/JabRef/jabref/blob/a76b8f3a3814b54cadcaad1826add136b33f2c98/src/main/java/org/jabref/gui/cleanup/FieldFormatterCleanupsPanel.java#L174-L177

You are right, one should usually not use model classes directly in the ui. So here the "correct" solution would be to wrap the `DatabaseMode` in a `DatabaseModeViewModel` which exposes the `getFormattedName` method. I feel like this is however overkill for this particular situation.